packages/salesforcedx-utils-vscode/test/jest/services/sourceTrackingService.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { SourceTracking } from '@salesforce/source-tracking';
+import { SourceTrackingService } from '../../../src/services';
+
+jest.mock('@salesforce/core', () => ({
+  ...jest.requireActual('@salesforce/core'),
+  Org: { create: jest.fn() },
+  SfProject: { resolve: jest.fn() }
+}));
+
+describe('Source Tracking Service', () => {
+  describe('createSourceTracking', () => {
+    let sourceTrackingCreateSpy: jest.SpyInstance;
+    let ensureLocalTrackingSpy: jest.SpyInstance;
+
+    beforeEach(() => {
+      sourceTrackingCreateSpy = jest
+        .spyOn(SourceTracking, 'create')
+        .mockResolvedValue({} as any);
+      ensureLocalTrackingSpy = jest
+        .spyOn(SourceTracking.prototype, 'ensureLocalTracking')
+        .mockResolvedValue({} as any);
+    });
+
+    it('Should create an instance of SourceTracking', async () => {
+      await SourceTrackingService.createSourceTracking('', {} as any);
+
+      expect(sourceTrackingCreateSpy).toHaveBeenCalled();
```

Comment:
One more test for should update?

Reply:
See [f32acb2](https://github.com/forcedotcom/salesforcedx-vscode/pull/4773/commits/f32acb268decc4dcf957266a7fc3c0e429386a87)